opus/application/static_media/js/utils.js

```diff
@@ -32,3 +32,28 @@ var o_utils = {
 
 
 };
+
+
+
+/**
+ * returns true if an element is visible, with decent performance
+ * @param [scope] scope of the render-window instance; 
+ * @returns {boolean}
+ */
+$.fn.isOnScreen = function(scope){
```

Comment:
Add space before {
Is there a reason to assign this to element when you could just use this throughout?

Reply:
I used code from another source and thought it interesting that they did it that way - I hadn't gotten to 'personalizing' the code other than seeing how it worked for me.